src/allocator.rs

```diff
@@ -702,8 +702,18 @@ fn test_point_atom_pair(#[case] fun: TestFun, #[case] expected: &str) {
 
 #[cfg(test)]
 #[rstest]
-#[case("97f1d3a73197d7942695638c4fa9ac0fc3688c4f9774b905a14e3a3f171bac586c55e83ff97a1aeffb3af00adb22c6bb")]
-#[case("a572cbea904d67468808c8eb50a9450c9721db309128012543902d0ac358a62ae28f75bb8f1c7c42c39a8c5529bf0f4e")]
+#[case(
+    "\
+97f1d3a73197d7942695638c4fa9ac0f\
+c3688c4f9774b905a14e3a3f171bac58\
+6c55e83ff97a1aeffb3af00adb22c6bb"
+)]
+#[case(
+    "\
+a572cbea904d67468808c8eb50a9450c\
+9721db309128012543902d0ac358a62a\
+e28f75bb8f1c7c42c39a8c5529bf0f4e"
+)]
```

Comment:
using the `hex_literal` `hex!` macro would be ideal here, but that's a bigger change